pkg/terminal/proxy.go

```diff
@@ -25,6 +25,8 @@ const (
 	ProxyEndpoint = "/api/terminal/proxy/"
 	// AvailableEndpoint path used to check if functionality is enabled
 	AvailableEndpoint = "/api/terminal/available/"
+	// InstalledNamespaceEndpoint path used to get the namespace where the controller is installed
+	InstalledNamespaceEndpoint = "/api/terminal/installedNamespace"
```

Comment:
Do we really need two of `AvailableEndpoint` and `InstalledNamespaceEndpoint`.
Since backend uses exactly the same logic, maybe client should call something like `/api/terminal/available`, `/api/terminal/operator`, `/api/terminal/operator-namespace` (resolves the operator's namespace or return 404 if it's not installed)

Reply:
I'm OK with doing either, the only reason why I'm hesitant to make the change is that we would have to modify useCloudShellAvailable.ts and that's called in multiple other non web terminal packages so it would require additional testing/review in those packages

Reply:
then maybe let's move it out of this PR.
Anyway someday we need to rename cloud-shell stuff to terminal, because it's really outdated terms )